projects/dotcms-field-elements/src/components/dot-checkbox/dot-checkbox.tsx

```diff
@@ -88,33 +81,29 @@ export class DotCheckboxComponent {
     }
 
     render() {
-        let labelTagParams: DotLabel = {
-            name: this.name,
-            label: this.label,
-            required: this.required
-        };
         return (
             <Fragment>
-                {getTagLabel(labelTagParams)}
-                {this._options.map((item: DotOption) => {
-                    const trimmedValue = item.value.trim();
-                    labelTagParams = { name: trimmedValue, label: item.label };
-                    return (
-                        <Fragment>
-                            <input
-                                class={getErrorClass(this.isValid())}
-                                type="checkbox"
-                                disabled={this.disabled || null}
-                                id={trimmedValue}
-                                checked={this.value.indexOf(trimmedValue) >= 0 || null}
-                                onInput={(event: Event) => this.setValue(event)}
-                                value={trimmedValue}
-                            />
-                            {getTagLabel(labelTagParams)}
-                        </Fragment>
-                    );
-                })}
-                {getTagHint(this.hint)}
+                <dot-label label={this.label} required={this.required} name={this.name}>
```

Comment:
why not just change the 'getTagLabel'?

Reply:
Because we can't use `<slot />` in a function.